db/fixtures/miq_event_definitions.csv

```diff
@@ -159,3 +159,9 @@ request_service_start,Service Start Request,Default,service_process
 service_started,Service Started,Default,service_process
 request_service_stop,Service Stop Request,Default,service_process
 service_stopped,Service Stopped,Default,service_process
+
+#
+# Container Operations
+#
+containerimage_scan_complete,Container Image Analysis Complete,Default,container_operations
+containerimage_created,New Container Image created,Default,container_operations
```

Comment:
This event would normally be called before starting the operation to allow policy a chance to cancel it.

Examples:
https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_or_template/scanning.rb#L37
https://github.com/ManageIQ/manageiq/blob/master/app/models/host.rb#L1349
https://github.com/ManageIQ/manageiq/blob/master/app/models/storage.rb#L161